orchagent/neighorch.cpp

```diff
@@ -1203,6 +1242,21 @@ void NeighOrch::voqSyncAddNeigh(string &alias, IpAddress &ip_address, const MacA
     sai_attribute_t attr;
     sai_status_t status;
 
+    Port inbandPort;
+    if (!gPortsOrch->getInbandPort(inbandPort))
+    {
+        SWSS_LOG_ERROR("Failed to get inband port/VLAN");
+        return;
+    }
+
+    // Inband neighbors are not synced to CHASSIS_APP_DB for VLAN type inband interface
+    // since these neigibors are learned within inband VLAN.
+    if (inbandPort.m_type == Port::VLAN && inbandPort.m_alias == alias)
+    {
```

Comment:
what is this case only for inband vlan ? Can you please explain which case it falls as per https://github.com/Azure/SONiC/blob/master/doc/voq/voq_hld.md ?

Reply:
In inband vlan case,  each asic (precisely its cpu port) participates in inband vlan. Each asic has its own inband IP address that belongs to the same inband vlan network. Each asic learns the inband IPs of other asics as neighbors. These neighbors are mainly used for the communication between SONiC instances like routing protocol peering.  Therefor, we added this change to not add inband neighbors into system neigh table.  This is described in inband vlan section (2.5.2) in voq HLD.

Reply:
thanks